packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -516,3 +516,36 @@ export const updateTemplateInstanceDescriptorDocumentErrorMapper = (
       () => HTTP_STATUS_NOT_FOUND
     )
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const addEServiceInterfceByTemplateErrorMapper = (
```

Comment:
```suggestion
export const addEServiceTemplateInstanceInterfaceErrorMapper = (
```

Reply:
handled with [c212232](https://github.com/pagopa/interop-be-monorepo/pull/1620/commits/c212232e6615a52174a8b71226d1609061ae7316)